_guide/_pages/people/2016-Assessment-Guide.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: 18F Engineering 2016 End of Year Assessment Guide
```

Comment:
do we need both of these assessments? should they be updated? has the system changed?

Reply:
looking through this, it actually has a bunch of good information so i am going to generalize the 2017 one and take out dates but leave it in.